### PR TITLE
added tests, formatting, wording

### PR DESCRIPTION
**Pull request by nilsbehlen**
* Added tests for PrivacyIDEA.cs
* Moved the submit button right below the otp input field

### CODE REVIEW

**Comment by nilsbehlen** [comment]
#3

File: Tests/SDKTests.cs [hunk 1]
```diff
@@ -0,0 +1,259 @@
...
+                                "    \"messages\": [\n" +
+                                "      \"Bitte geben Sie einen OTP-Wert ein: \",\n" +
+                                "      \"Please confirm the authentication on your mobile device!\"\n" +
+                                "    ],\n" +
+                                "    \"multi_challenge\": [\n" +
+                                "      {\n" +
+                                "        \"attributes\": null,\n" +
+                                "        \"message\": \"Bitte geben Sie einen OTP-Wert ein: \",\n" +
+                                "        \"serial\": \"OATH00020121\",\n" +
+                                "        \"transaction_id\": \"02659936574063359702\",\n" +
```
Comment (cornelinux) [inline Tests/SDKTests.cs:182]: You can already add the `client_mode` in the response here.

**Review by cornelinux — CHANGES_REQUESTED** [review]

File: Tests/SDKTests.cs (hunk 1 above)
Comment (nilsbehlen) [reply Tests/SDKTests.cs:182]: I dont think it makes sense to add this here prematurely when it is not processed by the code yet.

Comment (cornelinux) [reply Tests/SDKTests.cs:182]: OK.

**Review by cornelinux — APPROVED** [review]

**Pull request merged**